abs.yaml

```diff
@@ -0,0 +1,5 @@
+aggregate_check: false
```

Comment:
Hmm, I dislike those zombie recipes in AncondaRecipes. Actually by policy all recipes being imported to this repository should be pin either in aggregate/aggregateR/aggregate-deprecated/etc.
Otherwise we loose track about packages in defaults, and packages being not build for defaults, and their corresponding recipes.

Another note about this abs.yaml file here.  I understand why it is mentioned, but checking in fixed channel names is something just for intermediate to get out such a package, but need later on to be removed from feedstock.  Mainly by the reason that next time someone wants to build this feedstock will upload to this channel, without wanting, and additionally might get stuff from this channel, which is unwanted.

Reply:
@katietz thanks for the comment. Note that I do not intend to keep the `abs.yaml` file in the final build. I am probably doing the initial build wrong here, but I tend to develop in the services channel initially to ensure the build is correct. Once that passes, I remove abs.yaml, add it to the aggregate, and so forth. I apologize if I'm not doing things in the right order.

Reply:
Per instructions I've added libdate to the aggregate and removed abs.yaml altogether